src/components/MapSearch.tsx

```diff
@@ -4,15 +4,30 @@ import Constants from 'expo-constants';
 import {Container, Text, Icon, LineItem} from 'ScoutDesign/library';
 import {backArrow, searchThin} from 'ScoutDesign/icons';
 
+type MapSearchProps = {
+  locationToken: string,
+  searchText: string,
+  setSearchText: (text: string) => any,
```

Comment:
Oh yeah, I put it as `any` because that's really a callback for the parent - like it informs the parent that the searchText has changed. So it doesn't matter to the `MapSearch` itself what type the function returns.